.github/scripts/add_footer.py

```diff
@@ -0,0 +1,92 @@
+from os import walk
```

Comment:
If you're on Unix, could you make this executable please?

If you're on Windows, I can happily do it.

Reply:
It'll need the #! line too... you can copy it from one of the other files, such as update_releases.py...

Reply:
I run from WSL, so I'm both windows and Unix 😂 Nothing stops you from adding a shebang line and committing, right?

Reply:
> I run from WSL, so I'm both windows and Unix 😂 Nothing stops you from adding a shebang line and committing, right?

Only courtesy of not stepping on your toes!!!

Reply:
And wanting to share the good practice, as my impression is that you probably care about this sort of thing! :-)